addons/website_discuss_room/models/chat_room.py

```diff
@@ -53,9 +50,23 @@ def _compute_is_full(self):
             else:
                 room.is_full = room.participant_count >= int(room.max_capacity)
 
-    def _compute_jitsi_server_domain(self):
-        jitsi_server_domain = self.env['ir.config_parameter'].sudo().get_param(
-            'website_jitsi.jitsi_server_domain', 'meet.jit.si')
-
+    @api.depends("discuss_channel_id.rtc_session_ids")
+    def _compute_participant_count(self):
         for room in self:
-            room.jitsi_server_domain = jitsi_server_domain
+            room.participant_count = len(room.discuss_channel_id.rtc_session_ids)
+
+    def get_room_url(self):
+        self.ensure_one()
+        if not self.discuss_channel_id:
+            self._get_or_create_discuss_channel()
```

Comment:
you can directly do `channel = self._get_or_create_discuss_channel` and use the channel below as we basically do the same check in that method